src/js/edu-benefits/1990e/config/form.js

```diff
@@ -31,7 +48,36 @@ const formConfig = {
       pages: {
         applicantInformation: {
           path: 'applicant-information',
-          title: 'Applicant information'
+          title: 'Applicant information',
+          initialData: {},
+          uiSchema: {
+            applicantFullName: fullName.uiSchema,
+            applicantSocialSecurityNumber: ssn.uiSchema,
+            applicantDateOfBirth: currentOrPastDate.uiSchema('Applicant date of birth'),
```

Comment:
I've been using the language in the GH issues for each page, or what we've used for these fields in other forms (mainly the 1990). 

It's not a huge deal either way, since Molly and Alex will make a pass through the form and make copy changes (or tell us what changes to make).